hack/build.sh

```diff
@@ -66,6 +66,8 @@ for dir in ${dirs}; do
     *)
       if [ "${OS}" == "centos7" ]; then
         NAMESPACE="centos/"
+      elif [ "${OS}" == "fedora" ]; then
+        NAMESPACE="fedora/"
```

Comment:
Tested that this would work:

           NAMESPACE=$(grep "^FROM " "${dir}/Dockerfile.fedora" | cut -f2 -d/)/